atlp-rwanda/e-comm-team-brogrammers-fn
# #184948154 Setup GitHub Actions to run tests

#### What does this PR do?

this PR is to set up github actions to run test on push and pull requests on this repo

#### Description of Task to be completed?

is to setup github workflow to run tests and Add a coverage Badge in Readme

#### How should this be manually tested?

1. go to GitHub 
2. open the `repo` https://github.com/atlp-rwanda/e-comm-team-brogrammers-fn
3. check on the readme you will see the `covarage badge`

#### Any background context you want to provide?

n/a

#### What are the relevant pivotal tracker/Trello stories?

https://www.pivotaltracker.com/story/show/184948154

#### Screenshots (if appropriate)

<img width="844" alt="Screen Shot 2023-04-20 at 11 06 49" src="https://user-images.githubusercontent.com/118928495/233317743-6bed2bc3-4d07-4909-b5bc-5689ae378773.png">


#### Questions:
